lib/active_merchant/billing/gateways/paymentez.rb

```diff
@@ -292,9 +293,13 @@ def headers
       end
 
       def success_from(response)
-        return false if response.include?('error')
+        return false if response.include?('error') || FAILURE_STATUS.include?(response['status'])
```

Comment:
This should also be updated to look for current_status

Reply:
Noted!! thanks!